### PR TITLE
Preferred search language for TMDB

### PR DESCRIPTION
**Pull request by AMoo-Miki**
Part of an effort to bring in search lang, caching (where missing), tmdb urls in NFO files and standardizing the settings appearance.

### CODE REVIEW

**Comment by mkortstiege** [comment]
@olympia mind handling?

File: metadata.universal/resources/language/English/strings.po [hunk 1]
```diff
@@ -260,7 +260,11 @@ msgctxt "#30307"
 msgid "Enabling All Movie Categories will greatly reduce the search accuracy !!!"
 msgstr ""
 
-#empty strings from id 30308 to 30399
+msgctxt "#30308"
+msgid "     Preferred Search Language"
```
Comment (MartijnKaijser) [inline metadata.universal/resources/language/English/strings.po]: Remove the whitespaces

File: metadata.universal/resources/settings.xml [hunk 2]
```diff
@@ -102,6 +102,7 @@
     <setting type="lsep" label="30303"/>
     <setting type="sep"/>
     <setting label="30301" type="bool" id="tmdbsearch" default="false" enable="!eq(2,true)"/>
+    <setting label="30308" type="select" values="bg|cs|da|de|el|en|es|fi|fr|he|hr|hu|it|ja|ko|nl|no|pl|pt|ru|sl|sv|tr|zh" id="tmdbsearchlanguage" default="en" visible="!eq(-1,false)"/>
```
Comment (MartijnKaijser) [inline metadata.universal/resources/settings.xml]: Add  subsetting="true"
Why  visible="!eq(-1,false)" ? 
Use  visible="eq(-1,true)"

File: metadata.universal/resources/language/English/strings.po (hunk 1 above)
Comment (AMoo-Miki) [inline metadata.universal/resources/language/English/strings.po]: The whitespace is in par to what existed for IMDB lang

File: metadata.universal/resources/settings.xml (hunk 2 above)
Comment (AMoo-Miki) [inline metadata.universal/resources/settings.xml]: Merely copy-pasted the setting which exists for IMDB.

File: metadata.universal/resources/language/English/strings.po (hunk 1 above)
Comment (MartijnKaijser) [inline metadata.universal/resources/language/English/strings.po]: It's written and should be changed

Comment (AMoo-Miki) [inline metadata.universal/resources/language/English/strings.po]: I agree, especially considering "subsetting" exists for that purpose. I will introduce "subsetting" in another PR shortly.

**Pull request merged**